tests/unit/cluster/slot-migration.tcl

```diff
@@ -511,3 +511,34 @@ start_cluster 3 6 {tags {external:skip cluster} overrides {cluster-node-timeout
         wait_for_slot_state 2 ""
     }
 }
+
+
+proc log_file_matches {log pattern} {
```

Comment:
Is there a reason you re-implemented this function? It seems to exist elsewhere?

Reply:
I think we have a few copies of it, I can move it to util.tcl instead of copying it

Reply:
I guess we have count_log_message already there - just going to use that